auth: dnspcap2protobuf linker error on EL6 under devtoolset-4

- Program: Authoritative
 - Issue type: Bug report

### Short description
Auth with protobuf support doesn't seem to link fully on EL6 w/ devtoolset-4.

### Environment
 - Operating system: EL6
 - Software version: devtoolset-4, pdns auth 4.1.1
 - Software source: compiled myself

### Steps to reproduce
Build under EL6 environment with devtoolset-4

```
devtoolset-4-libstdc++-devel-5.3.1-6.1.el6.x86_64
devtoolset-4-runtime-4.1-3.sc1.el6.x86_64
devtoolset-4-gcc-5.3.1-6.1.el6.x86_64
devtoolset-4-gcc-c++-5.3.1-6.1.el6.x86_64
devtoolset-4-binutils-2.25.1-8.el6.x86_64
```

### Expected behaviour
Compiles and links fully

### Actual behaviour
```dnscap2protobuf``` errors out during linking with:

```
/opt/rh/devtoolset-4/root/usr/libexec/gcc/x86_64-redhat-linux/5.3.1/ld: gettime.o: undefined reference to symbol 'clock_gettime@@GLIBC_2.2.5'
//lib64/librt.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[3]: *** [dnspcap2protobuf] Error 1
```

### Other information
Manually linking with ```-lrt``` added seems to fix the problem. Habbie said on IRC that devtoolset-3 builds don't experience this.

```
$ grep lrt config.log
configure:18401: g++ -o conftest -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 --param ssp-buffer-size=4 -fstack-protector -Wl,-R/usr/lib64/pdns-auth -std=gnu++11  -L/home/dkinzel/git/powerdns/build/BUILD/pdns-auth-4.1.1/boost_1_66_0/stage/lib conftest.cpp -lrt   >&5
configure:18418: result: -lrt
ac_cv_search_clock_gettime=-lrt
RT_LIBS='-lrt '
```

Somewhere I am guessing this needs RT_LIBS tacked on?

```
dnspcap2protobuf_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CXX \
  $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=link $(CXXLD) \
  $(AM_CXXFLAGS) $(CXXFLAGS) $(dnspcap2protobuf_LDFLAGS) \
  $(LDFLAGS) -o $@
```

--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -1179,7 +1179,8 @@ dnspcap2protobuf_LDFLAGS = \
 dnspcap2protobuf_LDADD = \
 	$(LIBCRYPTO_LIBS) \
 	$(PROTOBUF_LIBS) \
-	$(BOOST_PROGRAM_OPTIONS_LIBS)
+	$(BOOST_PROGRAM_OPTIONS_LIBS) \
+	$(RT_LIBS)
 endif
 endif
 

